medusa/image_cache.py

```diff
@@ -157,6 +157,14 @@ def which_type(self, path):
         if not ek(os.path.isfile, path):
             logger.log(u"Couldn't check the type of " + str(path) + " cause it doesn't exist", logger.WARNING)
             return None
+        
+        if ek(getsize, path) == 0:
```

Comment:
@ratoaq2 ok to pass getsize as we already `import os.path` or we need to pass the full command?